h/streamer/streamer.py

```diff
@@ -89,7 +89,7 @@ def supervise(greenlets):
         gevent.joinall(greenlets, raise_error=True)
     except (KeyboardInterrupt, SystemExit):
         raise
-    except:  # noqa: E722
+    except:  # noqa: E722 # pylint:disable=bare-except
```

Comment:
Noted on the issue to remember to remove all of the pep8 `noqa` after switching to pylint.